### ISSUE TITLE
Use equalizer on a block grid

### ISSUE THREAD

**Issue by zzplab** [zurb/foundation-sites-6#221]
In F5 equalizer did not work on a block grid. I've used the js tweak for it.
I was hoping it would be possible in F6, but I checked and didn't work yet. Will that be done in the future?

**Comment by rafibomb** [comment]
I'm seeing it work out of the box on the docs - https://notebooks.zurb.com/posts/10023976?t=f7d4c8e6d071e640

Can you share you markup so we can see what's happening?

**Comment by zzplab** [comment]
In the fluid grid it works fine. But not in the block grid.
This file is in dir ../foundation6/testing/my-own-test-file.html where the ../foundation6/testing/scss/app.scss is compiled and used:
```
<!doctype html>
<!--[if IE 9]><html class="lt-ie10" lang="en" > <![endif]-->
<html class="no-js" lang="en" dir="ltr">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta class="foundation-mq">
    <title>Foundation for Sites 6.0.0</title>

    <link href="scss/sass.php/app.scss" rel="stylesheet" />
    <!-- <link rel="stylesheet" href="../node_modules/motion-ui/dist/motion-ui.css" /> -->
    <!-- uncomment this line ^ to use the motion-ui library -->
    <style>
	    .txtxt {
		   background: pink;
	    }
	    
	    </style>
  </head>
  <body>
          
  <div class="row small-up-1 medium-up-2 large-up-4" data-equalizer>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x400" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x300" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x300" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x200" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x300" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x500" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x300" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x200" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x300" class="thumbnail" alt="">
  </div>
  <div class="column txtxt" data-equalizer-watch>
    <img src="//placehold.it/300x500" class="thumbnail" alt="">
  </div>
</div>
    <script src="js/app.js"></script>
    <script src='../node_modules/motion-ui/dist/motion-ui.js'></script>
    <!-- uncomment this line ^ to use the motion-ui library -->
  </body>
</html>
```
Live url in direct mail preferably if you want.
![schermafbeelding 2015-11-07 om 00 57 02](https://cloud.githubusercontent.com/assets/7559497/11011643/897ee732-84ea-11e5-8a38-8c481db7e5ec.png)

**Comment by gakimball** [comment]
Not sure when this was fixed, but as of Foundation 6.0.4 the example you posted is working fine:

![image](https://cloud.githubusercontent.com/assets/1142001/11510654/9e5f50dc-9819-11e5-9d38-d494eab7195f.png)

We're tracking all future issues at [zurb/foundation-sites](https://github.com/zurb/foundation-sites), so if you run into any more problems, let us know there.

**Comment by zzplab** [comment]
I can confirm that. Great! Thanks.
